src/NexusMods.App.UI/Overlays/Updater/UpdaterViewModel.cs

```diff
@@ -74,7 +67,7 @@ public async Task<bool> MaybeShow()
     {
         if (!await ShouldShow()) return false;
 
-        _overlayController.SetOverlayContent(new SetOverlayItem(this));
+        _overlayController.Enqueue(this);
```

Comment:
Use `OverlayController` from parent class instead of having a private field here.